integrations/cross_platform/yocto.rst

```diff
@@ -0,0 +1,271 @@
+your Yocto builds.
+
+.. important::
+
+    We strongly recommend to use the Yocto's SDK toolchain to create packages as they will be built with the optimization flags suitable to
+    be deployed later to an image generated in a Yocto build.
+
+Deploying an application to a Yocto image
+=========================================
+
+Once you have created and and uploaded the Conan packages to a remote in Artifactory, you can deploy them in a Yocto build.
+
+Prepare your recipes
+--------------------
+
+It is important that in order to accomplish this, that the recipe of the application to be deployed should have a
+`deploy() method <https://docs.conan.io/en/latest/devtools/running_packages.html>`_. There you can specify the files of the application
+needed in the final image as well as any other from its dependencies (like shared libraries or assets):
+
+.. code-block:: python
+
+    def deploy(self):
+        self.copy("*", src="bin", dst="bin")
+        self.copy("*.dll", src="bin", dst="bin")
+        self.copy("*.dylib*", src="lib", dst="bin")
+        self.copy("*.so*", src="lib", dst="bin")
+        self.copy_deps("*.dll", src="bin", dst="bin")
+        self.copy_deps("*.dylib*", src="lib", dst="bin")
+        self.copy_deps("*.so*", src="lib", dst="bin")
+
+Set up the Conan layer
+----------------------
+
+We have created a [meta-conan](LINK) layer that includes all the configuration, the Conan client and a
+generic BitBake recipe. To add the layer you will have to clone it repository and the dependency layers of ``meta-openembedded``:
+
+.. code-block:: bash
+
+    $ cd poky
+    $ git clone LINK
+    $ git clone --branch thud https://github.com/openembedded/meta-openembedded.git
+
+You would also have to activate the layers in the *bblayers.conf* file of your build folder:
+
+.. code-block:: text
+   :caption: *conf/bblayers.conf*
+
+    POKY_BBLAYERS_CONF_VERSION = "2"
+
+    BBPATH = "${TOPDIR}"
+    BBFILES ?= ""
+
+    BBLAYERS ?= " \
+    /home/username/poky/meta \
+    /home/username/poky/meta-poky \
+    /home/username/poky/meta-yocto-bsp \
+    /home/username/poky/meta-openembedded/meta-oe \
+    /home/username/poky/meta-openembedded/meta-python \
+    /home/username/poky/meta-conan \
+    "
+
+After that, a Conan recipe to deploy a Conan package should look as easy as this Bitbake recipe:
+
+.. code-block:: text
+   :caption: *conan-mosquitto_1.4.15.bb*
+
+    inherit conan
+
+    DESCRIPTION = "An open source MQTT broker"
+    LICENSE = "EPL-1.0"
+
+    CONAN_PKG = "mosquitto/1.4.15@bincrafters/stable"
+    CONAN_REMOTE = "ARTIFACTORY_CONAN_REPOSITORY_URL"
+
+You will have to specify
+
+This recipe will be placed inside your own application layer that should be also added to the *conf/bblayers.conf* file.
+
+Build configuration
+-------------------
+
+Additionally to the recipe, you will need to provide the information about the credentials for Artifactory or the profile to be used to
```

Comment:
Artifactory for doing what? explain

Reply:
I have added an image with the flow and explained this a little bit more